gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.12.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-bin.zip
```

Comment:
Is it required change for this PR?

Reply:
I couldn't build this project without Gradle 8.13. Rolling back to 8.12.1 causes random errors in IntelliJ IDEA CE v2024.3.5